app/mapping/UserAnswersExtractor.scala

```diff
@@ -43,15 +43,22 @@ class UserAnswersExtractorImpl @Inject()(
 
   override def extract(answers: UserAnswers, data: GetTrust): Either[PlaybackExtractionError, UserAnswers] = {
 
+    val isTrustTaxable: Boolean = data.trust.details.trustTaxable match {
+      case Some(false) => false
+      case _ => true
+    }
```

Comment:
Can this be simplified to `!data.trust.details.trustTaxable.contains(false)` ?

Reply:
agreed, updated it now